.github/workflows/cicd-website.yml

```diff
@@ -0,0 +1,92 @@
+name: CI/CD Website branch
+
+on:
+  push:
+    branches:
+      - website
+  pull_request:
+    branches:
+      - website
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Set up Node.js version
+        uses: actions/setup-node@v3.2.0
+        with:
+          node-version: "16.13.x"
+
+      - name: yarn install
+        run: |
+          npm install -g yarn
+          yarn install --frozen-lockfile
+
+      - name: run linters
+        run: |
+          yarn lint:css
+          yarn lint:package
+          yarn lint:js
+          yarn lint:prettier
+
+      - name: build library
+        run: yarn build:library
+
+      - name: typecheck storybook
+        run: yarn typecheck:storybook
+
+      - name: build storybook
+        run: yarn build:storybook
+
+  publish-npm:
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.ref == 'refs/heads/website'
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GH_ADMIN_TOKEN }}
+
+      - name: Set up Node.js version
+        uses: actions/setup-node@v3.2.0
+        with:
+          node-version: "16.13.x"
+
+      - name: yarn install
+        run: |
+          npm install -g yarn
+          yarn install --frozen-lockfile
+
+      - name: build library
+        run: yarn build
+
+      - name: lerna version
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_ADMIN_TOKEN }}
+          GIT_AUTHOR_EMAIL: ${{ secrets.GIT_AUTHOR_EMAIL }}
+          GIT_AUTHOR_NAME: "Den Haag website"
+          GIT_COMMITTER_EMAIL: ${{ secrets.GIT_COMMITTER_EMAIL }}
+          GIT_COMMITTER_NAME: "Den Haag website"
+        run: |
+          git push --set-upstream origin HEAD
+          yarn release -- --yes
+
+      - name: lerna publish
+        with:
+          node-version: "16.13.x"
+          cache: "npm"
+          registry-url: "https://npm.pkg.github.com/"
+          scope: "@gemeente-denhaag"
+        env:
+          NPM_ACCESS_TOKEN: ${{ secrets.NPM_ACCESS_TOKEN }}
```

Comment:
You probable need to use another access token for this, or perhaps GitHub has some built in permissions for this?

Reply:
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
Yes! As discussed for now I'll use the NPM_ACCESS_TOKEN from NLDS CI bot, later we'll change it to a new one with permissions in the .yml